### DISCUSSION TITLE
Filtering Question

### DISCUSSION THREAD

**Discussion by adarshSrivastava01** [pacocoursey/cmdk#312] [Q&A]
How can I filter on the basis of the text inside the command item and not on the basis of value prop in there.

![image](https://github.com/user-attachments/assets/1fa4bb90-19ed-498d-9fe3-a462da2fde29)

I have a CommandInput through which I enter the searchQuery but it is filtering by the value prop, and not by the label prop.

**Comment by EdoAPP** [comment]
You can pass in the `label` to the `keywords` prop to CommandItem if the label is a string. 

```
<CommandItem value={option.value} keywords={[option.label]} />
```

**Reply by adarshSrivastava01** [reply]
yes it works this way too, or in my case specific all I had to do was to pass, `shouldFilter={false}`
